### PR TITLE
Fix integration tests

### LINKED ISSUE #432: Failing tests: docker image removed
### What happened?

```
Status: Downloaded newer image for osixia/openldap:1.3.0
Pulling keycloak (jboss/keycloak:10.0.2)...
The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.
```

### Example

FAILURE https://github.com/pulumi/pulumi-keycloak/pull/431

### Output of `pulumi about`

N/A

### Additional context

N/A

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

### PR DESCRIPTION
**Pull request by iwahbe**
Fixes #432 

This PR uses upstream's test setup to create a local testing keycloak harness. The deleted test files were copy and pasted from upstream originally, so nothing significant has changed in their contents, excepting that they were updated for keycloak's modern API paths.

### CODE REVIEW

File: .github/workflows/run-acceptance-tests.yml [hunk 1]
```diff
@@ -379,17 +416,17 @@ jobs:
...
       run: make install_${{ matrix.language}}_sdk
     - name: Install gotestfmt
       uses: GoTestTools/gotestfmt-action@v2
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
         version: v2.5.0
+    - name: Ensure upstream
+      run: make upstream
+    - name: Setup local Keycloak
+      run: make -C upstream local
```
Comment (VenelinMartinov) [inline .github/workflows/run-acceptance-tests.yml:429]: I don't see a `local` make target - where is that coming from?

Comment (iwahbe) [reply .github/workflows/run-acceptance-tests.yml:429]: The `-C` flag tells make to execute in that directory, so `make -C upstream local` is largely equivalent to `cd upstream && make local; cd -`. The `local` target is coming from upstream's makefile.

**Review by VenelinMartinov — APPROVED** [review]
lgtm

**Comment by VenelinMartinov** [comment]
Should we un-skip the tests too?

**Comment by iwahbe** [comment]
> Should we un-skip the tests too?

Which tests are skipped? The tests run. The repo hasn't run a successful PR in the last month.

**Pull request merged**